actor/v7action/route.go

```diff
@@ -228,9 +229,14 @@ func (actor Actor) GetRouteSummaries(routes []resources.Route) ([]RouteSummary,
 		}
 	}
 
-	spaces, _, ccv3Warnings, err := actor.CloudControllerClient.GetSpaces(ccv3.Query{
-		Key:    ccv3.GUIDFilter,
-		Values: uniqueSpaceGUIDs,
+	var spaces []resources.Space
+	ccv3Warnings, err := batcher.RequestByGUID(uniqueSpaceGUIDs, func(guids []string) (ccv3.Warnings, error) {
+		batch, _, warnings, err := actor.CloudControllerClient.GetSpaces(ccv3.Query{
```

Comment:
Just wondering why spaces are not included when getting the routes in the first place. It can be added according to docs. Not here though, I guess this is just a very nice refactor.